geometry_msgs/Pose pose
geometry_msgs/Twist velocity
humanoid_catching/Contact[] contacts
duration time
